keyboards/mixi/mixi.c

```diff
@@ -3,6 +3,7 @@
 void eeconfig_init_kb(void) {
 #ifdef RGBLIGHT_ENABLE
     rgblight_enable(); // Enable RGB underglow by default
+    rgblight_sethsv_noeeprom(0, 255, 255); // Set RGB default color
```

Comment:
You want to use a non `_noeeprom` function here given this callback is for setting the default eeprom config

```suggestion
    rgblight_sethsv(0, 255, 255); // Set RGB default color
```

Reply:
Applied at new commit.